backend/frontend/templates/frontend/user/board/partial_card.html

```diff
@@ -1,180 +1,201 @@
-                </div>
-                <div class="font-bold bg-blue-600 text-white rounded-full flex justify-center items-center h-5 px-2 text-xs -translate-x-1/2 -translate-y-2/4 top-0 right-0">{{card.open_pr_count}}</div> 
+    <!-- Card status and PR info section-->
+    <div class="grid grid-cols-2 gap-y-2 w-full h-[auto]">
+      <div class="col-span-1">
+        {% if card.code_review_competent_since_last_review_request %}
+          <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+            <i class="fa fa-smile-o" aria-hidden="true" title="Competent reviews"></i>
+            <div class="absolute inline-flex items-center justify-center w-5 h-5 text-xs text-white bg-red-600 border-2 border-white rounded-full -top-2 -end-2">
+              {{ card.code_review_competent_since_last_review_request }}
             </div>
+          </div>
         {% endif %}
+        {% if card.code_review_ny_competent_since_last_review_request %}
+          <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+            <i class="fa fa-frown-o"
+               aria-hidden="true"
+               title="Not yet competent reviews"></i>
+            <div class="absolute inline-flex items-center justify-center w-5 h-5 text-xs text-white bg-red-600 border-2 border-white rounded-full -top-2 -end-2">
+              {{ card.code_review_ny_competent_since_last_review_request }}
+            </div>
+          </div>
+        {% endif %}
+        {% if card.code_review_red_flag_since_last_review_request %}
+          <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+            <i class="fa-regular fa-face-sad-tear"
+               aria-hidden="true"
+               title="Red-flag reviews"></i>
+            <div class="absolute inline-flex items-center justify-center w-5 h-5 text-xs text-white bg-red-600 border-2 border-white rounded-full -top-2 -end-2">
+              {{ card.code_review_red_flag_since_last_review_request }}
+            </div>
+          </div>
+        {% endif %}
+        {% if card.code_review_excellent_since_last_review_request %}
+          <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+            <i class="fa-regular fa-face-laugh-wink"
+               aria-hidden="true"
+               title="Excellent reviews"></i>
+            <div class="absolute inline-flex items-center justify-center w-5 h-5 text-xs text-white bg-red-600 border-2 border-white rounded-full -top-2 -end-2">
+              {{ card.code_review_excellent_since_last_review_request }}
+            </div>
+          </div>
+        {% endif %}
+      </div>
+      {% if card.open_pr_count %}
+        {% with modified_url=card.repo_url|slice:"25:-4" %}
+          <a href="https://github.com/umuzi-org/{{ modified_url }}/pulls"
```

Comment:
I think it's best if we reuse the `get_github_repo_link` method that was used before and just append "/pulls" at the end and avoid hardcoding the URL here.